microsite/data/plugins/github-pull-requests-board.yaml

```diff
@@ -0,0 +1,9 @@
+---
+title: Github Pull Requests Board
+author: DAZN
+authorUrl: https://engineering.dazn.com/
+category: Source Control Mgmt
+description: View All open GitHub pull requests owned by your team in Backstage.
+documentation: https://github.com/backstage/backstage/tree/master/plugins/github-pull-requests-board
+iconUrl: img/github-pull-requests-board-logo.svg
```

Comment:
Is the logo created specifically for this plugin? Just want to make sure that we don't step on any toes with potentially copyrighted icons

Reply:
Yep, I created myself 😅